### PR TITLE
EL8 updates to succeed centos8 package builds, fix some dockerfile templating issues

### PR DESCRIPTION
**Pull request by punkrokk**
- EL8 dockerfiles had a some mixups between py2 and py3 causing rpmbuild command to fail. Defaulting to py2 to get everything working. Can consider py3 once py2 is working for packaging
- Fixed a templating error where the `/home/busybee/` folder was getting created with brackets eg.: `[/home/busybee/]`
- Other packages needs for EL8

### CODE REVIEW

File: packagingbuild/Dockerfile.template [hunk 1]
```diff
@@ -136,13 +136,13 @@ RUN PATH=/usr/share/python/st2python/bin:$PATH bash -c \
 {%- elif dist in ('centos', 'fedora') -%}
 
 # Install development tools and python for EL8
-RUN yum -y install python{%- if (version == '8') -%}3 python2 python3{%- endif %}-devel rpmdevtools && \
+RUN yum -y install python{%- if (version == '8') -%}3 python2 python2-devel gcc python3 {%- endif%}-devel rpmdevtools && \
```
Comment (armab) [inline packagingbuild/Dockerfile.template]: Looks like this template would generate `python3 -devel`

Comment (armab) [reply packagingbuild/Dockerfile.template]:
```suggestion
RUN yum -y install python{%- if (version == '8') -%}3 python2 python2-devel gcc python3{%- endif %}-devel rpmdevtools && \
```

Comment (punkrokk) [reply packagingbuild/Dockerfile.template]: It actually didn't matter, but I'm pushing an update to address it.

File: packagingbuild/centos8/Dockerfile [hunk 2]
```diff
@@ -33,14 +33,17 @@ RUN yum -y install \
...
+    libcurl-devel \
     libevent-devel \
     libffi-devel \
     libjpeg-devel \
     libtool \
     libwebp-devel \
     libxml2-devel \
-    libxslt-devel \make \
+    libxslt-devel \
+
```
Comment (armab) [inline packagingbuild/centos8/Dockerfile]: Would be good fixing templating to avoid blank line here

Comment (punkrokk) [reply packagingbuild/centos8/Dockerfile]: Fixed.

File: packagingbuild/centos8/Dockerfile [hunk 3]
```diff
@@ -85,13 +87,13 @@ RUN yum -y install nc net-tools
 #
 
 # Install development tools and python for EL8
-RUN yum -y install python3 python2 python3-devel rpmdevtools && \
+RUN yum -y install python3 python2 python2-devel gcc python3-devel rpmdevtools && \
```
Comment (armab) [inline packagingbuild/centos8/Dockerfile]: `gcc` is already installed before, - see line `33` of the same Dockerfile. See the template file as well.

I guess it somehow appeared here by copy-paste or whatever else similar reason.

**Comment by punkrokk** [comment]
Happy New Year!

**Pull request merged**